ee/clickhouse/models/event.py

```diff
@@ -284,6 +285,14 @@ def get_event_count_month_to_date() -> int:
     return result
 
 
+def get_billable_event_count_last_30_days() -> int:
```

Comment:
An additional approach would be to pull the bill information from Stripe (bridged through license.posthog.com) like we do on cloud (https://github.com/PostHog/posthog-cloud/blob/49c5101b56d31b06b1612e1c3f4c2bfbe0f48a69/multi_tenancy/stripe.py#L183), that way the customer can also now what's their current bill up to today.